operation.go

```diff
@@ -330,7 +329,7 @@ func (o *Operation) Password(prompt string) ([]byte, error) {
 	o.t.EnterRawMode()
 	defer o.t.ExitRawMode()
 
-	b, err := terminal.ReadPassword(int(os.Stdin.Fd()))
+	b, err := terminal.ReadPassword(int(o.cfg.Stdin.Fd()))
```

Comment:
Oh, my comment was after looking at the source for terminal. They literally just set `t.echo = false` then read a line:

https://github.com/golang/crypto/blob/master/ssh/terminal/terminal.go#L649